### ISSUE TITLE
Option to add a continuation marker at the front of a text chunk.

### ISSUE THREAD

**Issue by Dark-Cr0w** [LadyDefile/Wordsmith-DalamudPlugin#33]
This may be a niche suggestion but it would be nice if there was the option to input something to put at the front of chunks from the second chunk onward.
Example being:

/em [chunk one] **>>**
/em *||* [chunk two] *>>*
/em *||* [chunk three]

**Comment by LadyDefile** [comment]
# Thank you!
Love the suggestion. I'm always open to more if you come up with something else so don't be shy! I want Wordsmith to be the best it could possibly ever be. Even if you think it may be niche. I enjoy the challenge of implementing new features and fixing bugs.

# About the suggestion
Honestly, I was thinking of expanding on the continuation marker system already. I hadn't before because there were already so many options but now that I've got them sorted into categories I may revisit the idea. I just have to decide how far I want to go with it.

I'll definitely keep this suggestion in mind. I'll take some time today and play around with it. I want to be sure to do it right so I may or may not have some kind of prototype by the end of the day. I'll definitely update you with the ideas I come up with though.

I've got a couple ideas that I would like to play around with and see how it goes but I don't want to run my mouth about what may or may not end up existing just yet so just know that I am working on it!

# TLDR
Thank you so much for suggesting! I'll work on it.

**Comment by LadyDefile** [comment]
# Update
Hello again. I just wanted to give you an update on the solution I've come to. I've added an entirely new `Marks & Tags` category to Scratch Pad settings tab. It looks like this:
![image](https://user-images.githubusercontent.com/67084868/208534535-adf33ba0-1f36-4e40-8fa9-806fecb9ea3f.png)
As you can see, the OOC tags and Continuation Marker have both been moved to this new section. But the exciting part (and relevant part to this topic) is the `New Marker` section at the bottom.

In the above example I put your \|\| mark in and this is it in action in a Scratch Pad.
![image](https://user-images.githubusercontent.com/67084868/208537475-8a36947d-ab08-4dc0-8d07-03cece350ff6.png)

So now you can do something like this:
![image](https://user-images.githubusercontent.com/67084868/208539002-ce0c4a3d-fa73-4dde-8280-e682556966d2.png)



## Another placeholder
I'm adding a `#r` placeholder for marker text that will place the number of chunks remaining

## New Marker
* `Marker Text` Is the text to be written. Yes `#c`, `#m`, and `#r` work here.
* `Position` Before OOC opening tags, before text body, after text body, after OOC closing tags, and after continuation marker.
* `Show On` lets you choose when/where to show. such as `All except first` or `only on Last`
* `Display when there is/are _____ chunks` lets you make it display based on if there are chunks or not
* `Display when OOC is _____` lets you specify if it show up when OOC is on, off, or either.

## Final Notes
I'm not sure when this update will go live but I wanted you to know that I'm working on it. I was a little worried that this new system would be overly complicated so I did stick it behind the `Show Advanced Settings` option in the `General` settings tab. This is a pretty complicated overhaul on the way chunks are calculated so I've got to do some testing and really make sure this doesn't completely fudge something else up. And yes, I said 5 positions but there are only 4 options there. I'm in the middle of adding the 5th one (after continuation mark)

I hope you keep supporting and enjoying Wordsmith and I'll try to get this out to as soon as I'm able.

**Comment by LadyDefile** [comment]
# Ready For Testing!
The update is looking good. I've fixed a couple bugs while working on all of this. The final form of the new marker system looks like this: 
![image](https://user-images.githubusercontent.com/67084868/208592463-1ed0c1e9-e52d-40ce-88b0-5116c0e94d51.png)
You can use the arrows on the left to reorganize the list (with some limitations. You can't move something above/below it's group. For instance, in the example image, you couldn't move `(03)` above `(10)` or below `(04)`

I should be able to push this out as a testing version in the next day or two!
